### PR TITLE
ЧеремушкинКИ Лаб2

### PR DESCRIPTION
**Pull request by CheremshkinKI**

### CODE REVIEW

**Comment by CheremshkinKI** [comment]
@Azariashvili
@BrazhnikDA

File: modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class Polynom : public Monom {
+ private:
+    Monom* StartMonom;
+    int SIZE = 1;
+ public :
+    Polynom();
+    explicit Polynom(int size);
+    Polynom(const Polynom& polynom);
+
+    void DeleteList();
```
Comment (BrazhnikDA) [inline modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h:16]: Rename function ClearList

File: modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 Cheremushkin Kirill
+#ifndef MODULES_POLYNOM_AND_MONOM_CHEREMUSHKINKI_INCLUDE_POLYNOM_H_
+#define MODULES_POLYNOM_AND_MONOM_CHEREMUSHKINKI_INCLUDE_POLYNOM_H_
+#pragma once
+#include "Monom.h"
```
Comment (BrazhnikDA) [inline modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h:5]: Change path

File: modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h (hunk 1 above)
Comment (CheremshkinKI) [reply modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h]: corrected

File: modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h (hunk 2 above)
Comment (CheremshkinKI) [reply modules/Polynom_and_Monom_CheremushkinKI/include/Polynom.h:5]: They are in the same folder, so you shouldn't write the path

File: modules/Polynom_and_Monom_CheremushkinKI/include/Monom.h [hunk 3]
```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Cheremushkin Kirill
+#ifndef MODULES_POLYNOM_AND_MONOM_CHEREMUSHKINKI_INCLUDE_MONOM_H_
+#define MODULES_POLYNOM_AND_MONOM_CHEREMUSHKINKI_INCLUDE_MONOM_H_
+#pragma once
+#include <iostream>
+class Monom {
```
Comment (Azariashvili) [inline modules/Polynom_and_Monom_CheremushkinKI/include/Monom.h:6]: Please comment on the class

Comment (CheremshkinKI) [reply modules/Polynom_and_Monom_CheremushkinKI/include/Monom.h:6]: corrected

**Comment by CheremshkinKI** [comment]
@SHVlader

**Review by BrazhnikDA — APPROVED** [review]

**Pull request merged**